### ISSUE TITLE
[Windows] Final Fantasy XIV seems to run in exclusive fullscreen even when set to borderless

### ISSUE THREAD

**Issue by JaneSmith** [doitsujin/dxvk#2640]
I'm playing Final Fantasy XIV. It uses DirectX 11 and is set to borderless fullscreen mode. There are various third-party programs that provide helpful information for this game in the form of on-screen overlays.

I recently started using DXVK, and was very pleased to see that it has helped resolve some issues I was having with DirectX 11. However, I've also noticed that none of my overlays are working anymore. It is as if the game is running in exclusive fullscreen mode, instead of borderless mode.

I did a search around, and I found the old issue #1397. As far as I can understand from this, setting the game to borderless should work, but it isn't working.

### Software information

Final Fantasy XIV, set to use DirectX 11 and borderless fullscreen mode.

### System information

- GPU:  AMD Radeon RX 5700 XT
- Driver: 21.50.21.07 and/or 30.0.15021.7000
- Wine version: N/A
- DXVK version: 1.10.1

### Apitrace file(s)

- Not sure this is applicable as I'm trying to use this on Windows rather than Linux.

### Log files

- Not sure this is applicable as I'm trying to use this on Windows rather than Linux.

**Comment by doitsujin** [comment]
This is not really a bug, Vulkan doesn't really have the concept of exclusive fullscreen mode the same way that DXGI does, so drivers on Windows tend to "optimize" Vulkan applications into exclusive fullscreen mode because it gives them a slightly faster presentation path.

TL;DR won't/can't fix, especially since this is a complete non-issue on Linux.

**Comment by vlad54rus** [comment]
In the dxvk.conf file try setting dxgi.tearFree = True and dxgi.syncInterval = 0

**Comment by AFNGP** [comment]
Facing the same issue on another MMO called Soulworker. Borderless Windowed runs Exclusive Fullscreen. I have apps showing on top of the game generally usually (i.e. a DPS parser) and Exclusive Fullscreen shows on top of such apps no matter what.

Would be interesting if someone found a workaround to the problem. Will try the one proposed by @vlad54rus once I get home and report back.

**Comment by JaneSmith** [comment]
Thank you for the suggestion. Unfortunately, I tried it and it didn't make any difference. I agree, it would be great if there were a workaround to this. But if it isn't possible, then I understand.

For the record, I am a Linux user, but I'm still stuck on a Windows virtual machine for my gaming as one of my games still uses anti-cheat software that refuses to work on Linux. Thanks for all your hard work on this.

**Comment by vlad54rus** [comment]
@AFNGP Are you on AMD or NVIDIA? Do you still have this issue?

**Comment by bluenote73** [comment]
@vlad54rus I'm on AMD, and tried those settings, overlays such as volume control didn't work. Happy to try if you have other suggestions, ty
